apps/E2E/src/Button/specs/Button.spec.win.ts

```diff
@@ -53,24 +69,32 @@ describe('Button Functional Testing', () => {
   });
 
   it('Validate OnClick() callback was fired -> Click', async () => {
-    await ButtonPageObject.clickComponent();
-    await expect(await ButtonPageObject.didOnClickCallbackFire()).toBeTruthy();
+    await ButtonPageObject.click(ButtonPageObject._primaryComponent);
+    await expect(await ButtonPageObject.waitForOnClickCallbackToFire()).toBeTruthy(
+      'Mouse click failed to fire the onClick callback for the primary button.',
```

Comment:
Nit: Let's re-word this a bit:

"Clicking on the primary button failed to fire the onClick() callback"